app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/dashboard.tsx

```diff
@@ -2,19 +2,21 @@
  * interactive components. */
 import * as react from 'react'
 
-import * as projectManagerModule from 'enso-content/src/project_manager'
-
-import * as auth from '../../authentication/providers/auth'
-import * as backend from '../service'
+import * as backendApi from '../cloudBackendApi'
```

Comment:
1. Why are we changing the name from `backend` to `backendApi`? Every file defines API, so in fact, every file could be named `...Api`. 
2. It should be imported as `cloudBackend`, as it's now confusing with normal Enso's backend.